modules/markup/csv/csv.go

```diff
@@ -104,7 +104,7 @@ func (Renderer) Render(ctx *markup.RenderContext, input io.Reader, output io.Wri
 	}
 
 	rd, err := csv.CreateReaderAndGuessDelimiter(bytes.NewReader(rawBytes))
-	if err != nil {
+	if err != nil && err != io.EOF {
```

Comment:
I would worry that in this case the rd would be nil and we'd get an NPE below.

Reply:
That's actually why I also have to not return err if the err is io.EOF and let it do the CreateReader. The data is an empty byte array so it just defaults to comma as the delimiter.

Reply:
Ok, changed it...does that seem better?

Reply:
yeah better!